kasa/cli/discover.py

```diff
@@ -14,9 +14,17 @@
     Discover,
     UnsupportedDeviceError,
 )
-from kasa.discover import ConnectAttempt, DiscoveryResult
+from kasa.discover import (
+    NEW_DISCOVERY_REDACTORS,
+    ConnectAttempt,
+    DiscoveredRaw,
```

Comment:
Just a question, the result is stored in `DiscoveryResult`, should we follow the same naming scheme for the raw data, i.e., name this `DiscoveryResultRaw`?

Reply:
I think this is ok because the `Result` is just part of the raw. There is also the `error_code` in the raw (and potentially something else but who knows 🤷 ) which is at the same level as the result.